js/src/admin/components/MailPage.js

```diff
@@ -149,10 +169,56 @@ export default class MailPage extends Page {
     return this.fields.some((key) => this.values[key]() !== app.data.settings[key]);
   }
 
+  sendTestEmail() {
+    if (this.saving || this.sendingTest()) return;
+
+    this.sendingTest = m.prop(true);
+    const settings = {};
+
+    this.fields.forEach((key) => (settings[key] = this.values[key]()));
```

Comment:
If I understand correctly, the saved settings from the database are now used to send the test. So this means this code is no longer needed (?)